apps/app/src/contexts/AuthContext.tsx

```
@@ -1,55 +1,443 @@
+        if (authMethod === "web3auth" && web3AuthInstanceFromHook) {
+          const authResult = await web3AuthInstanceFromHook.authenticateUser();
+          return authResult.idToken;
+        } else if (authMethod === "near" && nearAccountDetails?.accountId) {
+          return nearAccountDetails?.accountId;
+          return await sign({
+            signer: near,
+            recipient: "curatefun.near", // Replace if different
+            message:
+              purposeMessage ||
+              `authentication for ${nearAccountDetails.accountId}`,
+          });
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)